### PR TITLE
[DOCS-7614] Eppo feature flag integration

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do? What is the motivation?

- Adds a new section for the Eppo feature flag integration

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by drichards-87 — APPROVED** [review]
Left you a couple of suggestions and approved the PR. (I see you marked the PR as merge after reviewing, but it looks like you're waiting on reviews from at least one other person. So I'll let you merge it.)

**Comment by rtrieu** [comment]
@drichards-87 Thanks DeForest, appreciate the review!

**Pull request merged**
